react-datetime.d.ts

```diff
@@ -50,17 +50,17 @@ declare module ReactDatetime {
      only parameter, if the date in the input is valid. If it isn't, the value
      of the input (a string) is returned.
      */
-    onChange?:(x: string) => void;
+    onChange?:(x: string|any) => void;
```

Comment:
I believe the documentation is phrased in a way that leads to confusion. If we look through source code, we see that the result of `onChange` is never used. The documentation string is taken from the root `README.md` file. I've amended that part of the configuration as well. Ref: `DateTime.js:184`.